app/Http/Controllers/RegisterController.php

```diff
@@ -47,8 +47,8 @@ public function loginUser(Request $request){
                 $accountant = Accountant:: where('mail','=',$request->mail)->first();
                 if($accountant){
                     if (Hash::check($request->password, $accountant->password)){
-                        $request->session()->put('loginId',$accountant->id);
-                        return redirect('dashboard');
+                        $request->session()->put('login_Id',$accountant->id);
```

Comment:
```suggestion
                        $request->session()->put('login_id',$accountant->id);
```
Use `snake_case` for session key

Reply:
Make similar changes throughout

Reply:
Thanks for the feedback sir, I'll update code in the latest commit.